src/index.html

```diff
@@ -1,13 +1,68 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+          href="https://fonts.gstatic.com" 
+          crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body class="page">
+
+    <div class="card" data-qa="card">
+      <img src="images/imac.jpeg" 
+           alt="images" 
+           class="card__image"
+      >
+
+      <div class="card__texts">
+        <h2 class="card__texts-text"> 
+          Моноблок APPLEА1419 iMAC 27" Retina 5K (MNED2UA/A)
+        </h2>
+      </div>
+  
+      <div class="card__small-text">
+        <p class="card__small-textcode">Код товара: 195434</p>
+      </div>
+
+    </div>
+
+    <div class="star">
+
+      <div class="stars stars--4">
+        <div class="stars-star"></div>
+        <div class="stars-star"></div>
+        <div class="stars-star"></div>
+        <div class="stars-star"></div>
+        <div class="stars-star"></div>
+      </div>
+
+      <div class="star__text">
+        <p class="star__text-paragraf">Отзывов: 5</p>
+      </div>
+
+    </div>
```

Comment:
is it one star in `.star` block?)
```suggestion
    <div class="star">
      <div class="stars stars--4">
        <div class="stars-star"></div>
        <div class="stars-star"></div>
        <div class="stars-star"></div>
        <div class="stars-star"></div>
        <div class="stars-star"></div>
      </div>

      <div class="star__text">
        <p class="star__text-paragraf">Отзывов: 5</p>
      </div>
    </div>
```